tests/fixtures/issues/issue-539.php

```diff
@@ -1485,59 +1485,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-539-AM' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 6.1; en-US)  AppleWebKit/537.1 (KHTML, like Gecko) Chrome/21.0.1180.83 Safari/537.1',
```

Comment:
This one has two spaces before "AppleWebKit", which the generic pattern will still get, but the versioned patterns won't.